views/administration/school.hbs

```diff
@@ -426,7 +427,7 @@
 								</option>
 								{{/each}} </select> </div> {{/if}}
 						{{> "lib/components/csrfInput"}}
-						<button type="submit" class="btn btn-primary btn-submit">{{$t "administration.school.button.saveGeneralSettings" }}</button>
+						<button type="submit" class="btn btn-primary btn-submit" data-testid="school-admin-save-general-setting">{{$t "administration.school.button.saveGeneralSettings" }}</button>
```

Comment:
see above

Reply:
here updated the name to -> 'school-administration-save-general-setting'